index.html

```diff
@@ -596,12 +628,33 @@ <h3>Terminology</h3>
           <dt><dfn class="export">verification relationship</dfn></dt>
           <dd>
             <p>
-An expression of the relationship between a [=subject=] and a
+An expression of the relationship between an [=identifier=] and a
 [=verification method=]. One example of a verification relationship is
```

Comment:
This doesn't seem quite right to me. An identifier is just a string / symbol. These relationships are between the subject (as identified by the base identifier) and the verification method. What is being said by verification relationship R is that the associated verification method VM can be used to verify a proof for a particular purpose on behalf of the subject *when it is identified by the identifier X*.

Also note the change from "the relationship" to "a relationship" since there can be more than one. I'd prefer to just keep "a [=subject=]" because the wording was simple enough for me, but if this more wordy expression gets to consensus I'm ok with it (or something like it):

```suggestion
An expression of a relationship between a [=subject=], when
identified by the [=base identifier=], and a [=verification method=].
One example of a verification relationship is
```

Reply:
I disagree with this characterization, but I want to acknowledge it's a difference of worldview that has some irreducible impedence mismatch. Which is to say I understand where Dave is coming from, but I disagree that that perspective is helping us.

We cannot know for certain who the subject referred to by the identifier is. Determining the subject referred to by an identifier is an output after the verification and validation of claims made in reference to that identiifier. As such, it is a mistake to START with the presumption that the verification method applies to the subject. It might. It might not.

What we do know is that particular verification methods are declared as suitable for verifying proofs from the controller of the identifier. Whoever that controller happens to be, these verification methods can be applied to have cryptographic confidence that the proof was created by the controller.

The subject may not be involved at all.

Reply:
> We cannot know for certain who the subject referred to by the identifier is.

What matters is *which* subject it is. And the ID is what allows there to be no confusion about *which one*. The fact that the ID is "controllable" is what allows there to be no confusion about whether certain proofs are verifiable by verification methods actually authorized by the controller to be used behalf of *this* subject. These are the only elements that matter about the subject at this layer. Talking about "who" or "what" the subject is here (e.g., "who is the subject?", "what real world thing is it?"), in my view, continues to be wholly irrelevant and is what I think creates this confusion.

Just because we say "this subject has these authorized verification methods" doesn't mean we also then have to debate whether the subject is a human being who likes rainy days. It also doesn't mean that if we can't decide whether that's true or not that we have to say that it's really the ID that has the verification methods, not the subject. It just doesn't follow. The ID just lets me know *which* subject -- and then I can use that information to figure out *which* verification methods are authorized for what (or which services I can use to interact with the subject).

> The subject may not be involved at all.

I think we disagree what a "subject" is. Your comments imply to me that your mental model includes more in its definition than is necessary and more than I believe the spec says. Again, as an example, I see no reason whatsoever at this layer to ask: "but what real world thing is the subject?"

In my view of a "subject", it is the centerpiece of this technology. The whole point of a controller document is to enable a controller to logically establish "a thing" (a subject) that can act, interact, etc. with other things -- and to provide a mechanism for others to be able to verify information about those actions/interacts and that they were authorized by the controller. The ID is what allows there to be no confusion about which subject that is (out of all possible subjects) and the fact that it is "controllable" is what allows there to be no confusion about whether proofs were made by authorized verification methods.

When I speak of a "subject" that's it, it's just a logical construct used to distinguish it from "other things". All that matters is that we're "referring to the same 'thing'" when we use the same ID and that we can get some information to interact with it or verify some stuff supposedly done on its behalf.

All other debates about whether that "same thing" has horns, is a good idea or a bad one, wears blue pants, or likes to jump are not relevant here.